corehq/motech/repeaters/migrations/0001_adjust_auth_field_format_squashed_0015_drop_connection_settings_fk.py

```diff
@@ -136,7 +136,7 @@ class Migration(migrations.Migration):
                 ('is_paused', models.BooleanField(default=False)),
                 ('next_attempt_at', models.DateTimeField(blank=True, null=True)),
                 ('last_attempt_at', models.DateTimeField(blank=True, null=True)),
-                ('connection_settings', models.ForeignKey(on_delete=django.db.models.deletion.PROTECT, to='motech.connectionsettings')),
+                ('connection_settings_id', models.IntegerField(db_index=True)),
```

Comment:
Is it possible _for a user_ to delete the ConnectionSettings of a Repeater?

Reply:
We do our best to prevent it, although to my knowledge it's now impossible to prevent it with raw SQL. That was implemented in https://github.com/dimagi/commcare-hq/pull/32850/commits/1514acde5b0730893cdb206cffc816e9ab18fd23